runtime/util/finish_rewriting.cpp

```diff
@@ -51,6 +51,7 @@ void init_outputs(char const *output_filename) {
   }
 
   if (statistics) {
+    int steps = get_steps();
```

Comment:
this should definitely not be an `int` because that can easily cause overflow issues. I would go with `uin64_t` here as well